src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -348,7 +352,9 @@ void BaseQNAcceleration::performAcceleration(
         "filter or increase its threshold (larger epsilon).");
   }
 
-  splitCouplingData(cplData); // split the primary and secondary coupling data back into the individual data objects
+  // Apply the quasi-Newton update to the waveform
+  splitCouplingData(cplData);
```

Comment:
For me the purpose has not changed, since we still need to split the _values back into the coupling data. The difference is that we now also update the waveform in the coupling data as well. I have updated the comment to something better reflect whats actually going on here. 
 
After some thought I changed the name to updateCouplingData. This sounds a little bit better in my opinion, since I am technically also changing the time grid of the waveforms in cplData. I am however, open for better suggestions, since this is still not very descriptive.

Reply:
In this case vaguer is more correct. So it's a decent choice. :+1: